augly/text/augmenters/typo.py

```diff
@@ -114,7 +115,11 @@ def __init__(
             assert (
                 misspelling_dict_path is not None
             ), "'misspelling_dict_path' must be provided if 'typo_type' is 'all' or 'misspelling'"
+            assert max_typo_length >= 1, "Must set 'max_typo_length' >= 1"
+            self.max_typo_length = max_typo_length
             self.model = self.get_model(misspelling_dict_path)
+        else:
+            self.max_typo_length = 1;
```

Comment:
```suggestion
            self.max_typo_length = 1
```
no semicolons in python!